controllers/gatekeeper_controller.go

```diff
@@ -236,14 +237,30 @@ func (r *GatekeeperReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	}
 
 	if err := r.handleCPSController(ctx, gatekeeper); err != nil {
-		if errors.Is(err, errCrdNotReady) {
+		if errors.Is(err, errCpsCrdNotReady) {
 			// ConstraintPodStatus CRD is not ready, wait for the CRD is ready
 			return ctrl.Result{RequeueAfter: time.Second * 3}, nil
 		} else {
 			return ctrl.Result{}, err
 		}
 	}
 
+	if err := r.handleConfigController(ctx); err != nil {
+		if errors.Is(err, errConfigCrdNotReady) {
+			// Config CRD is not ready, wait for the CRD is ready
+			return ctrl.Result{RequeueAfter: time.Second * 3}, nil
+		} else {
+			return ctrl.Result{}, err
+		}
+	}
+
+	err = r.initConfig(ctx)
```

Comment:
Is this required? Doesn't your new controller handle this already?

Reply:
Nothing trigger the reconcile so I need to create a config

Reply:
Oh right... sounds good.